src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -266,5 +267,58 @@ public override int GetHashCode()
                 Hash.Combine(base.GetHashCodeHelper(),
                 Hash.Combine((int)this.SpecifiedLanguageVersion, 0));
         }
+
+        /// <summary>
+        /// Parse a LanguageVersion from a string input, as the command-line compiler does.
+        /// </summary>
+        public static bool TryParseLanguageVersion(string version, out LanguageVersion result)
```

Comment:
>TryParseLanguageVersion [](start = 27, length = 23)

It is not clear why this public API is inside CSharpParseOptions class. I think it belongs closer to LanguageVersion type, next to ToDisplayString API.